### PR TITLE
Unit tests to ensure timeout calculated properly

### PR DESCRIPTION
**Pull request by ethanfrey**
There were some issues, let's track them down.
So far, the current code looks proper.

### CODE REVIEW

File: src/lib/utils.ts [hunk 1]
```diff
@@ -315,7 +314,7 @@ export function multiplyCoin({ amount, denom }: Coin, mult: number): Coin {
 }
 
 // return true if a > b, or a undefined
-function heightGreater(a: Height | undefined, b: Height): boolean {
+export function heightGreater(a: Height | undefined, b: Height): boolean {
```
Comment (bartmacbartek) [inline src/lib/utils.ts:317]:
```suggestion
export function heightGreater(a?: Height, b: Height): boolean {
```

File: src/lib/utils.ts [hunk 2]
```diff
@@ -333,7 +332,7 @@ function heightGreater(a: Height | undefined, b: Height): boolean {
 
 // return true if a > b, or a 0
 // note a is nanoseconds, while b is seconds
-function timeGreater(a: Long | undefined, b: number): boolean {
+export function timeGreater(a: Long | undefined, b: number): boolean {
```
Comment (bartmacbartek) [inline src/lib/utils.ts:335]:
```suggestion
export function timeGreater(a?: Long, b: number): boolean {
```

**Review by bartmacbartek — APPROVED** [review]
Looks good to me, approved with syntax suggestions.

File: src/lib/utils.ts (hunk 1 above)
Comment (ethanfrey) [reply src/lib/utils.ts:317]: Nope. You cannot have a required argument after an optional argument - tsc already caught me on this one (I tried it)

**Comment by ethanfrey** [comment]
Since `tsc` and `eslint` require the syntax I used, I will keep it (I do prefer your suggestions)

**Pull request merged**